account/verification_code.go

```diff
@@ -0,0 +1,189 @@
+package account
+
+import (
+	"context"
+	"time"
+
+	"github.com/fabric8-services/fabric8-auth/application/repository"
+	"github.com/fabric8-services/fabric8-auth/errors"
+	"github.com/fabric8-services/fabric8-auth/gormsupport"
+	"github.com/fabric8-services/fabric8-auth/log"
+
+	"github.com/goadesign/goa"
+	"github.com/jinzhu/gorm"
+	errs "github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+)
+
+type VerificationCode struct {
+	gormsupport.Lifecycle
+	ID     uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"` // This is the ID PK field
+	User   User
+	UserID NullUUID `sql:"type:uuid"`
```

Comment:
UserID can't be null, right? So UserID should be uuid.UUID

Reply:
Right, we dont need NullUUID , 
6d6d759